implementing-domain-logic/command-handling/aggregate-polymorphism.md

```diff
@@ -0,0 +1,45 @@
+# Aggregate Polymorphism
+
+In certain cases it is beneficial to have a polymorphic hierarchy in aggregate structure. Subtypes in polymorphic 
+aggregate hierarchy inherit `@CommandHandler`s, `@EventSourcingHandler`s and `@CommandHandlerInterceptor`s from the 
+super aggregates. Based on `@AggregateIdentifier` the correct aggregate type is loaded and command is executed on it. 
+Let's take a look at the following example:
+
+```java
+public class Parent {}
```

Comment:
Alongside this theoretical example, would it be possible to provide a variant on the `GiftCard` which also shows some of the method handlers? The `GiftCard` application is used as the sample project in the majority of the guide, hence such a sample would be beneficial for continuity when reading the guide.

Reply:
Yeah I know, I'll try to think of polymorphic cases in `GiftCard` example.

Reply:
Muchos gracias señor